src/NativeModule.web.js

```diff
@@ -0,0 +1,3 @@
+import KeyCommandWeb from './KeyCommand.web';
```

Comment:
Create a folder `KeyCommand`, move this file as a child named `index.web.js` and then move `NativeModule.native.js` as a second child named `index.native.js`.